app/domain/authentication/authn_jwt/identity_providers/validate_identity_configured_properly.rb

```diff
@@ -28,6 +29,7 @@ def validate_identity_configured_properly
           return unless identity_available?
 
           id_claim_key
```

Comment:
Is this call is necessary there
`validate_token_app_property_configured_properly` will call `id_claim_key` any way